src/assets/YAML/generated/generated.yaml

```diff
@@ -0,0 +1,8597 @@
+---
```

Comment:
The generated.yaml should not be part of this repo because it is generated in the -data repo